manifests/server/config.pp

```diff
@@ -115,6 +116,10 @@
     }
 
   }
+  # Allow timestamps in log by default
+  postgresql::server::config_entry {'log_line_prefix':
```

Comment:
Can you shield this by a `if $log_line_prefix` ? I'd expect that this creates an invalid config entry when $log_line_prefix is undef; also some people might want to configure that somewhere else.